baseline.yaml

```
@@ -125,6 +125,38 @@ criteria:
     control_mappings: # TODO
     scorecard_probe:
       - blocksDeleteOnBranches
+  - id: OSPS-TEST
```

I'm not sure if this is "quality" or "build and release".  In either case, you probably need a different ID:

```suggestion
  - id: OSPS-QA-08
```